ci_framework/roles/set_openstack_containers/tasks/kustomize_edpm.yml

```diff
@@ -0,0 +1,74 @@
+---
+- name: Make EDPM manifests directory exists
+  ansible.builtin.file:
+    path: "{{ cifmw_set_openstack_containers_basedir }}/artifacts//kustomizations/dataplane/"
+    state: directory
+
+- name: Construct container registry
+  ansible.builtin.set_fact:
+    _container_registry: "{{ cifmw_set_openstack_containers_registry }}/{{ cifmw_set_openstack_containers_namespace }}"
+
+- name: Construct container url
+  ansible.builtin.set_fact:
+    _container_url: "{{ _container_registry }}/{{ cifmw_set_openstack_containers_prefix }}"
+
+- name: Create EDPM Container CR customization
+  ansible.builtin.copy:
+    dest: "{{ cifmw_basedir }}/artifacts/manifests/kustomizations/dataplane/100-kustomization.yaml"
+    content: |-
+      apiVersion: kustomize.config.k8s.io/v1beta1
+      kind: Kustomization
+      resources:
+      namespace: {{ cifmw_install_yamls_defaults.NAMESPACE }}
+      patches:
+      - target:
+          kind: OpenStackDataPlaneNodeSet
+        patch: |-
+
+          - op: add
```

Comment:
wonder if we can have some jinja going over a loop here :) for all the edpm services to prevent the repetition here

but i think it would probably get messy to do it like that so meh just a thought